service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,360 @@
+            <parameter name="approvedDate" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Party" list="orderCreateList">
+                <econdition field-name="partyId" value="CustJqp"/>
+            </entity-find>
+            <if condition="orderCreateList">
+                <then>
+                    <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+                </then>
+            </if>
+        </actions>
+    </service>
+
+
+    <service verb="create" noun="CreateOrderPart" displayName="Create the order Part">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName" required="true"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" required="true"/>
+            <parameter name="customerPartyId" required="true"/>
+            <parameter name="item_details" type="List">
+                <parameter name="orderEntry" type="Map">
+                    <parameter name="productId"/>
+                    <parameter name="itemDescription"/>
+                    <parameter name="quantity"/>
+                    <parameter name="unitAmount"/>
+                </parameter>
+            </parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="orderPartSeqId" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderCreateList">
+                <econdition field-name="orderId"/>
+            </entity-find>
+            <if condition="orderCreateList">
+                <service-call name="create#mantle.order.OrderPart" in-map="context" out-map="context"/>
+            </if>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderList">
+                <econdition field-name="orderId"/>
+                <econdition field-name="customerPartyId"/>
+            </entity-find>
+            <set field="productId" from="item_details.productId"/>
+            <set field="unitAmount" from="item_details.unitAmount"/>
+            <entity-find entity-name="mantle.product.Product" list="productList">
+                <econdition field-name="productId"/>
+            </entity-find>
+            <iterate list="productList" entry="product">
+                <set field="productId" from="product.productId"/>
+            </iterate>
+            <iterate list="orderList" entry="order">
+                <set field="orderId" from="order.orderId"/>
+            </iterate>
+            <if condition="orderList &amp;&amp; productList">
+                <then>
+                    <service-call name="create#mantle.order.OrderItem"
```

Comment:
We need to create OrderItems for all the input items in the list, please check.

Reply:
Updated recommended changes.
Thank you.